build.gradle

```diff
@@ -23,7 +23,7 @@ buildscript {
 
         ext.jacocoAndroidVersion = "0.2"
 
-        ext.coverallsVersion = "2.12.0"
+        ext.coverallsVersion = "2.12.2"
```

Comment:
Noticed this version number did not exist in our libs.versions.toml file, so we should move it over.